packages/wasm/as/assembly/msgpack/Read.ts

```diff
@@ -48,4 +49,6 @@ export abstract class Read {
 
   abstract isNextNil(): bool;
   abstract isNextString(): bool;
+
```

Comment:
I'd like to add context messages when reading items from an array. Will make a push now with this added.